src/main/java/factionsplusplus/integrators/DynmapIntegrator.java

```diff
@@ -75,7 +75,7 @@ public DynmapIntegrator(
             try {
                 this.dynmapAPI = (DynmapCommonAPI) dynmap; /* Get API */
                 if (this.dynmapAPI == null) {
-                    this.logger.error("Instantiated DynmapCommonAPI object was null. DynmapIntegrator construction cannot continue.");
+                    this.logger.error("Instantiated DynmapCommonAPI object was null. DynmapIntegrator construction cannot continue.", null);
```

Comment:
might be worth overloading error() in Logger so it doesn't always require a second parameter